res/i18n/fr/labels.json

```diff
@@ -481,38 +481,38 @@
 		"enabled": "Activé",
 		"fade_in": "Fondu ouverture",
 		"fade_out": "Fondu fermeture",
-		"head_cut": "Écrêtage",
+		"head_cut": "Coupe du début",
 		"listen": "Écoute",
-		"makeup": "Surgain",
+		"makeup": "Gain",
 		"predelay": "Pré-délai",
 		"reverse": "Inverser",
 		"sample_num": "Échantillon #",
-		"tail_cut": "Équeutage"
+		"tail_cut": "Coupe de la fin"
 	},
 	"settings": "Paramètres",
 	
-	"side_direction": "Direction de côté", 
+	"side_direction": "Cotés directivité", 
```

Comment:
It sounds/looks very wrong to me.
Proposal: "_Direction latérale_".

Reply:
This parameter control the patern (8 or bidirectional) of the "side" channel of an Mid/Side signal.
So i think that we can't translate "direction" but "directivité" (for "pattern" from polar pattern). The "side" word is more difficult, but litteraly it's "cotés" (plurials : we have 2 sides in the channel), and i don't find better wihout make a full sentence. Maybe "Directivité des cotés" ?

Reply:
"Directivité des cotés" : OK for this one @wargreen.